clients/vscode-hlasmplugin/src/test/syntaxes/endevor/basics.listing.snap

```diff
@@ -179,9 +179,8 @@
 #                                                                                                                ^ source.hlasmListing keyword.other.hlasm
 #                                                                                                                 ^^^^^^^^^^^^^^^^^^^^ source.hlasmListing invalid.ignored
 >                                                       REMARKS                                                  1234567             
```

Comment:
Comment should start in column 16. The spaces before are not part of the comment. Not sure if it is worth fixing if that would be complicated.

Reply:
I've adjusted it in a way that multiline remarks always start at column 16 regardless if there are just spaces on the line. Was this the goal?

Reply:
Yes, perfect. Thank you!

Reply:
One more comment. Looking at line 178
```
#                                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasmListing
```
This should technically be two
```
#                                                            ^ source.hlasmListing
#                                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasmListing comment.hlasm
```

Reply:
Alright, adjusted 👍